providers/yttcb/clusterbootstrap.yaml

```diff
@@ -315,7 +362,13 @@ spec:
   additionalPackages:
     - refName: metrics-server*
     - refName: secretgen-controller*
+    #@ if pinniped_configs_exist():
     - refName: pinniped*
+      valuesFrom:
+        secretRef: #@ "{}-pinniped-addon".format(data.values.CLUSTER_NAME)
+    #@ else:
```

Comment:
This actually seems CONCERNING to us.
Given that a user might not configure pinniped with LDAP or OIDC initially, but may configure it later, I think we want a consistent output here of a certain specific secret name so a user can find it easily.

Reply:
We decided to let the `secretRef` use its default value for this PR.